private/buf/cmd/buf/command/dep/internal/internal.go

```diff
@@ -53,11 +53,12 @@ func ModuleKeysAndTransitiveDepModuleKeysForModuleRefs(
 
 // Prune prunes the buf.lock.
 //
-// Used by both mod prune and mod update.
+// Used by both dep/mod prune and dep/mod update.
```

Comment:
```suggestion
// Used by dep/mod prune.
```

It appears that update is no longer using this, so we can tweak the comment.